### PR TITLE
Plaintext votes 2

### PR DESCRIPTION
**Pull request by vteague**
This is a start on getting plaintext up- and down-voting to work. I also organised dismissed-votes a little better.

Up-voting only works on the QuestionDisplayCard, not on the QuestionDetailView. Down-voting is not implemented yet. On the QuestionDisplayCard, both numbers are the total number of upvotes.

Flagging is not implemented except to turn the flag red.

While testing my changes, I noticed Issue #140 .  However, I don't think this PR introduces it because it was present in main already.

### CODE REVIEW

File: RightToAskClient/UnitTests/Models/QuestionTests.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+using Xunit;
+
+namespace UnitTests.Models
+{
+    // For testing Question.cs
+
+    public class QuestionTests
+    {
+
+        /* FIXME - this test displays exactly the same unpredictable behaviour that was
```
Comment (LukeMcCarthy-TW) [inline RightToAskClient/UnitTests/Models/QuestionTests.cs:14]: Should we fix this test before merging?

File: RightToAskClient/UnitTests/QuestionViewModelTests.cs [hunk 2]
```diff
@@ -196,9 +194,13 @@ public void UpvoteCommandTest()
             executeAsyncButton(button);
 
             // assert
-            Assert.Equal(1, vm.Question.UpVotesByThisUser);
+            Assert.True(vm.Question.AlreadyUpvoted);
         }
 
+        /*
```
Comment (LukeMcCarthy-TW) [inline RightToAskClient/UnitTests/QuestionViewModelTests.cs]: We should delete this code and add a card to the back log

File: RightToAskClient/RightToAskClient/RightToAskClient/Models/QuestionResponseRecords.cs [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+     */
+    public class QuestionResponseRecords
+    {
+	    private HashSet<string> _upvotedQuestionIDs;
+	    private HashSet<string> _downvotedQuestionIDs;
+	    private HashSet<string> _reportedQuestionIDs;
+	    private HashSet<string> _dismissedQuestionIDs;
+	    private bool _isInitialised;
+
+		public void Init()
```
Comment (LukeMcCarthy-TW) [inline RightToAskClient/RightToAskClient/RightToAskClient/Models/QuestionResponseRecords.cs:23]: Indentation seems strange in this file, are the methods meant to be indented that far?

Comment (LukeMcCarthy-TW) [reply RightToAskClient/RightToAskClient/RightToAskClient/Models/QuestionResponseRecords.cs:23]: Don't worry about this

File: RightToAskClient/RightToAskClient/RightToAskClient/Models/QuestionResponseRecords.cs [hunk 4]
```diff
@@ -0,0 +1,101 @@
+using System;
```
Comment (LukeMcCarthy-TW) [inline RightToAskClient/RightToAskClient/RightToAskClient/Models/QuestionResponseRecords.cs:1]: Would like some more tests for this. For example try and test what happens when you try to add something in the hashset

**Review by LukeMcCarthy-TW — APPROVED** [review]

**Comment by vteague** [comment]
corrected error that caused tests to fail.

**Pull request merged**